src/backoffice/dashboards/examine-management/dashboard-examine-management.element.ts

```diff
@@ -1,17 +1,66 @@
-import { UUITextStyles } from '@umbraco-ui/uui-css/lib';
-import { css, html, LitElement } from 'lit';
-import { customElement } from 'lit/decorators.js';
+import { html, LitElement, css } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { IRoute, IRoutingInfo, path } from 'router-slot';
+
+import { UmbDashboardExamineIndexElement } from './views/section-view-examine-indexers';
+import { UmbDashboardExamineSearcherElement } from './views/section-view-examine-searchers';
+
+import { UmbContextConsumerMixin } from '@umbraco-cms/context-api';
 
 @customElement('umb-dashboard-examine-management')
-export class UmbDashboardExamineManagementElement extends LitElement {
-	static styles = [UUITextStyles, css``];
+export class UmbDashboardExamineManagementElement extends UmbContextConsumerMixin(LitElement) {
+	static styles = [
+		css`
+			a {
+				color: var(--uui-color-text);
+				background: transparent;
+				border: none;
+				text-decoration: underline;
+				cursor: pointer;
+				display: inline-block;
+			}
+		`,
+	];
+	@state()
+	private _routes: IRoute[] = [
+		{
+			path: `/index/:indexerName`,
+			component: () => import('./views/section-view-examine-indexers'),
+			setup: (component: HTMLElement, info: IRoutingInfo) => {
+				const element = component as UmbDashboardExamineIndexElement;
+				element.indexName = info.match.params.indexerName;
+			},
+		},
+		{
+			path: `/searcher/:searcherName`,
+			component: () => import('./views/section-view-examine-searchers'),
+			setup: (component: HTMLElement, info: IRoutingInfo) => {
+				const element = component as UmbDashboardExamineSearcherElement;
+				element.searcherName = info.match.params.searcherName;
+			},
+		},
+		{
+			path: ``,
+			component: () => import('./views/section-view-examine-overview'),
+		},
+	];
+
+	@state()
+	private _currentPath?: string;
+
+	private _onRouteChange() {
+		this._currentPath = path();
+	}
+
+	private get backbutton(): boolean {
+		return this._currentPath != '/section/settings/dashboard/examine-management/' || !this._currentPath ? true : false;
+	}
 
 	render() {
-		return html`
-			<uui-box>
-				<h1>Examine Management</h1>
-			</uui-box>
-		`;
+		return html` ${this.backbutton
+				? html` <a href="/section/settings/dashboard/examine-management/"> &larr; Back to overview </a> `
```

Comment:
I haven't actually toyed with it yet, but do we have a way to route relative to the current path, so we don't have to hardcode the entire URL in places such as this?

Reply:
I guess something like ${window.location.href}/index/${indexName}` would work when going from the overview and into the index view (and similar for searcher view), but then going back to the overview we would have to grab the URL and cut off the last part. I'm not sure if there are any other ways.

Reply:
This is going to be a TODO for the future...